modules/virt-booting-efi-mode.adoc

```diff
@@ -5,21 +5,21 @@
 [id="virt-booting-vms-efi-mode_{context}"]
 = Booting virtual machines in EFI mode
 
-You can configure a virtual machine to boot in EFI mode by editing the VM or VMI manifest.
+You can configure a virtual machine to boot in EFI mode by editing the VM manifest.
 
 .Prerequisites
 
 * Install the OpenShift CLI (`oc`).
 
 .Procedure
 
-. Create a YAML file that defines a VM object or a Virtual Machine Instance (VMI) object. Use the firmware stanza of the example YAML file:
+. Create a YAML file that defines a VM object. Use the firmware stanza of the example YAML file:
 +
 .Booting in EFI mode with secure boot active
 [source,yaml]
 ----
   apiversion: kubevirt.io/v1
-  kind: VirtualMachineInstance
+  kind: VirtualMachine
   metadata:
     labels:
       special: vmi-secureboot
```

Comment:
For the sake of consistency:

s/vmi-secureboot/vm-secureboot  

Applies to both the `special` and `name` fields